app/javascript/hyacinth_ui_v1/components/digital_objects/rights/rights_form/ItemRightsForm.js

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import produce from 'immer';
+import { Form, Collapse } from 'react-bootstrap';
+import { useHistory } from 'react-router-dom';
+import { merge } from 'lodash';
+
+import Label from '../../../ui/forms/Label';
+import InputGroup from '../../../ui/forms/InputGroup';
+import BooleanRadioButtons from '../../../ui/forms/inputs/BooleanRadioButtons';
+import { digitalObject as digitalObjectApi } from '../../../../util/hyacinth_api';
+import FormButtons from '../../../ui/forms/FormButtons';
+
+import DescriptiveMetadata from './subsections/DescriptiveMetadata';
+import CopyrightStatus from './subsections/CopyrightStatus';
+import CopyrightOwnership from './subsections/CopyrightOwnership';
+import ColumbiaUniversityIsCopyrightHolder from './subsections/ColumbiaUniversityIsCopyrightHolder';
+import LicensedToColumbiaUniversity from './subsections/LicensedToColumbiaUniversity';
+import ContractualLimitationsRestrictionsAndPermissions from './subsections/ContractualLimitationsRestrictionsAndPermissions';
+import RightsForWorksOfArtSculptureAndPhotographs from './subsections/RightsForWorksOfArtSculptureAndPhotographs';
+import UnderlyingRights from './subsections/UnderlyingRights';
+
+function ItemRightsForm(props) {
+  const { digitalObject } = props;
+  const { id, rights: initialRights, dynamicFieldData } = digitalObject;
+  const history = useHistory();
+
+  const defaultRights = () => ({
```

Comment:
Can we move this template object function into its own file?

Reply:
Sure!  It was already there before my refactor, so I was going for minimal change where possible, but I like the idea of having it in a different file. Will push a commit for this soon.

Reply:
Pushed!